Geometry/VeryForwardGeometry/src/CTPPSPixelTopology.cc

```diff
@@ -1,8 +1,5 @@
 #include "Geometry/VeryForwardGeometry/interface/CTPPSPixelTopology.h"
```

Comment:
Are the changes in this file necessary? (i.e. doing more than changing the code formatting?)

Reply:
Yes since scram b code-formats was required.